fs/react-next-base
# [257] Use ESLint fix command. Fix some styles

# Summary

#### A brief description of the pull request.

#### Resolves: #257 

#### Screenshots in case of UI changes
Before changes:
![Screen Recording 2022-06-14 at 13 27 00](https://user-images.githubusercontent.com/47859969/173612110-e741e5dc-099f-4351-bdd4-124f23aad505.gif)

After changes:
![Screen Recording 2022-06-14 at 13 27 39](https://user-images.githubusercontent.com/47859969/173612458-73f70437-6211-4e7e-859a-21efa1381fda.gif)

------
Style fixes
Before changes:
<img width="654" alt="Screenshot 2022-06-15 at 15 23 44" src="https://user-images.githubusercontent.com/47859969/173826001-eb81e46b-6bbf-4b75-a5bb-98ef4598cfb6.png">
<img width="649" alt="Screenshot 2022-06-15 at 15 23 36" src="https://user-images.githubusercontent.com/47859969/173826068-f6bf3c19-2f95-45d7-a206-c1dce4990339.png">



After changes:
<img width="674" alt="Screenshot 2022-06-15 at 15 20 07" src="https://user-images.githubusercontent.com/47859969/173825833-9a2bca5c-acc3-4286-b34d-32bf2bfcad47.png">
<img width="714" alt="Screenshot 2022-06-15 at 15 20 23" src="https://user-images.githubusercontent.com/47859969/173825855-8799b082-8c7b-45aa-9b9f-a9dfbc50f48d.png">

# Test plan

#### List of steps to manually test introduced functionality:

To review style fixes:
- Go to app locally, login
- Go to `/form-examples` page, you should see checkbox and textarea correctly
- Go to `/activity` page, by changing screen width you should see correct styles for user avatar in table 

#### What do need to pay more attention to, some implicit changes that influence the app:

- ...

# Review notes

#### While reviewing pull-request (especially when it's your pull-request), please make sure that:

- you named PR close to issue name and linked that issue
- you understand what problem is solved by PR and how is it solved
- new tests are in place, no redundant tests
- required ENV variables added and described in `.env.example` and added to Heroku
- associated Heroku review app works correctly with introduced changes
